src/main/java/com/resource/SendRetrieveTweets.java

```diff
@@ -0,0 +1,36 @@
+package com.resource;
+import org.eclipse.jetty.util.StringUtil;
+import twitter4j.TwitterException;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.util.ArrayList;
+@Produces(MediaType.APPLICATION_JSON)
+@Path("/api/1.0/twitter")
+public class SendRetrieveTweets {
```

Comment:
Confusing class name 
TwitterController

Reply:
name is changed